.jenkins/github/platform-management-sdk-tests.Jenkinsfile

```diff
@@ -0,0 +1,190 @@
+@Library('c8y-common-steps') _
+def INSTANCE_NAME
+def ADMIN_CREDENTIALS_ID
+pipeline {
+    agent {
+        kubernetes {
+            yaml '''
+        apiVersion: v1
+        kind: Pod
+        spec:
+          containers:
+          - name: "java"
+            image: "registry.stage.c8y.io/ci/jdk:11"
+            command: ["/bin/sh"]
+            args: ["-c", "cat"]
+            tty: true
+            env:
+            - name: JENKINS_URL
+              value: http://jenkins.ci.svc.cluster.local:8080/
+            - name: MAVEN_USER_HOME
+              value: /home/jenkins/agent
+            - name: DOCKER_HOST
+              value: tcp://localhost:2375
+            workingDir: "/home/jenkins/agent"
+            resources:
+              requests:
+                cpu: 2
+                memory: 3Gi
+              limits:
+                cpu: 3
+                memory: 3Gi
+        '''
+        }
+    }
+    options {
+        timeout(time: 8, unit: 'HOURS')
+        skipDefaultCheckout()
+    }
+    stages {
+        stage('Initialize for post-merge') {
```

Comment:
I wish we could put all that logic into a single common lib function call instead of duplicating it... Same for staging.

Reply:
I believe we can do that in separate task